Dockerfile

```diff
@@ -13,7 +13,12 @@ RUN cargo install --locked --path .
 
 FROM debian:buster-slim
 
-RUN apt update && apt install -y bash curl netcat tini wget
+RUN apt update && apt install -y ca-certificates
+
+#Let debian repos be fetched securely:
+RUN sed -i "s/http:\/\//https:\/\//g" /etc/apt/sources.list
```

Comment:
Can you explain more why this change was necessary?

Reply:
It wasn't completely necessary, but by default debian fetches from its repos using only http://